modules/preview-react/form-field/lib/FormFieldLabel.tsx

```diff
@@ -2,36 +2,19 @@ import React from 'react';
 
 import {createSubcomponent} from '@workday/canvas-kit-react/common';
 import {useFormFieldLabel, useFormFieldModel} from './hooks';
-import {createStencil, px2rem} from '@workday/canvas-kit-styling';
-import {mergeStyles} from '@workday/canvas-kit-react/layout';
+import {createStencil, parentModifier, px2rem} from '@workday/canvas-kit-styling';
+import {FlexProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {brand, system} from '@workday/canvas-tokens-web';
+import {formFieldStencil} from './formFieldStencil';
 
-export interface FormFieldLabelProps {
+export interface FormFieldLabelProps extends FlexProps {
   /**
    * The text of the label.
    */
   children: React.ReactNode;
-  /**
-   * Will style the text as disabled
-   */
-  disabled?: boolean;
-  /**
-   * Changes the color of the text
-   */
-  variant?: 'error' | 'hint' | 'inverse';
 }
 
-const formFieldLabelAsteriskStencil = createStencil({
-  base: {
-    fontSize: system.fontSize.body.large,
-    fontWeight: system.fontWeight.normal,
-    color: brand.error.base,
-    textDecoration: 'unset',
-    marginInlineStart: system.space.x1,
-  },
-});
-
-const formFieldLabelStencil = createStencil({
+export const formFieldLabelStencil = createStencil({
   base: {
     ...system.type.subtext.large,
```

Comment:
```suggestion
  extends: textStencil,
  base: {
    ...system.type.subtext.large,
```

Reply:
We can get rid of `...system.type.subtext.large`, right?